### PR TITLE
bump SHA

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: prow/proxy-common.inc [hunk 1]
```diff
@@ -43,15 +43,16 @@ fi
...
-
-  # Use RBE when logged in.
-  BAZEL_BUILD_RBE_INSTANCE="${BAZEL_BUILD_RBE_INSTANCE:-projects/istio-testing/instances/default_instance}"
-  BAZEL_BUILD_RBE_JOBS="${BAZEL_BUILD_RBE_JOBS:-200}"
-  if [[ -n "${BAZEL_BUILD_RBE_INSTANCE}" && "${BAZEL_BUILD_RBE_JOBS}" -gt 0 ]]; then
-    echo "Using RBE: ${BAZEL_BUILD_RBE_INSTANCE}"
-    export BAZEL_BUILD_ARGS="${BAZEL_BUILD_ARGS} --google_credentials=${GOOGLE_APPLICATION_CREDENTIALS} --config=remote-clang-libc++ --config=remote-ci --remote_instance_name=${BAZEL_BUILD_RBE_INSTANCE} --jobs=${BAZEL_BUILD_RBE_JOBS}"
-  fi
-fi
+# Disabling RBE due to failures like https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_proxy/2633/release-test_proxy/211
```
Comment (PiotrSikora) [inline prow/proxy-common.inc]: `export BAZEL_BUILD_RBE_JOBS=0` works as well, without the need to comment out everything.

Comment (PiotrSikora) [reply prow/proxy-common.inc]: Also, you could disable it only for the release build, and not all.

**Comment by kyessenov** [comment]
@jplevyak @PiotrSikora TSAN failure is legit:
```
WARNING: ThreadSanitizer: data race (pid=40998)
  Write of size 1 at 0x5636bdfc8a2a by main thread:
    #0 Envoy::Extensions::Common::Wasm::V8::engine() <null> (envoy+0x5b7ad23)
    #1 Envoy::Extensions::Common::Wasm::V8::V8::clone() <null> (envoy+0x5b7be66)
    #2 Envoy::Extensions::Common::Wasm::Wasm::Wasm(Envoy::Extensions::Common::Wasm::Wasm const&, Envoy::Event::Dispatcher&) <null> (envoy+0x5b2e974)
    #3 std::__1::__compressed_pair_elem<Envoy::Extensions::Common::Wasm::Wasm, 1, false>::__compressed_pair_elem<Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&, 0ul, 1ul>(std::__1::piecewise_construct_t, std::__1::tuple<Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>, std::__1::__tuple_indices<0ul, 1ul>) <null> (envoy+0x5b6bf6c)
    #4 std::__1::__compressed_pair<std::__1::allocator<Envoy::Extensions::Common::Wasm::Wasm>, Envoy::Extensions::Common::Wasm::Wasm>::__compressed_pair<std::__1::allocator<Envoy::Extensions::Common::Wasm::Wasm>&, Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>(std::__1::piecewise_construct_t, std::__1::tuple<std::__1::allocator<Envoy::Extensions::Common::Wasm::Wasm>&>, std::__1::tuple<Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>) <null> (envoy+0x5b6bd4a)
    #5 std::__1::__shared_ptr_emplace<Envoy::Extensions::Common::Wasm::Wasm, std::__1::allocator<Envoy::Extensions::Common::Wasm::Wasm> >::__shared_ptr_emplace<Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>(std::__1::allocator<Envoy::Extensions::Common::Wasm::Wasm>, Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&) <null> (envoy+0x5b6bbc1)
    #6 std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::Wasm> std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::Wasm>::make_shared<Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>(Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&) <null> (envoy+0x5b6b990)
    #7 std::__1::enable_if<!(is_array<Envoy::Extensions::Common::Wasm::Wasm>::value), std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::Wasm> >::type std::__1::make_shared<Envoy::Extensions::Common::Wasm::Wasm, Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&>(Envoy::Extensions::Common::Wasm::Wasm&, Envoy::Event::Dispatcher&) <null> (envoy+0x5b41624)
    #8 Envoy::Extensions::Common::Wasm::createThreadLocalWasm(Envoy::Extensions::Common::Wasm::WasmHandle&, std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::Plugin>, absl::string_view, Envoy::Event::Dispatcher&) <null> (envoy+0x5b332a6)
    #9 Envoy::Extensions::Common::Wasm::getOrCreateThreadLocalWasm(Envoy::Extensions::Common::Wasm::WasmHandle&, std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::Plugin>, absl::string_view, Envoy::Event::Dispatcher&) <null> (envoy+0x5b33c6e)
    #10 Envoy::Extensions::HttpFilters::Wasm::FilterConfig::FilterConfig(envoy::extensions::filters::http::wasm::v3::Wasm const&, Envoy::Server::Configuration::FactoryContext&)::$_0::operator()(std::__1::shared_ptr<Envoy::Extensions::Common::Wasm::WasmHandle>) const::'lambda'(Envoy::Event::Dispatcher&)::operator()(Envoy::Event::Dispatcher&) const <null> (envoy+0x4a76535)
```

**Comment by PiotrSikora** [comment]
Oh, I know what it is. Hold tight.

**Comment by PiotrSikora** [comment]
Fix is out: https://github.com/envoyproxy/envoy-wasm/pull/380 (it's harmless, btw).

**Comment by kyessenov** [comment]
If it's harmless, then something else must be causing occasional TSAN failures (when it doesn't segfault).

**Pull request merged**